app/controllers/admin/api/services_controller.rb

```diff
@@ -111,7 +113,12 @@ def can_create
   end
 
   def service_params
-    params.fetch(:service)
+    params.require(:service).permit(:name, :system_name, :description, :support_email, :deployment_option, :backend_version,
```

Comment:
Those are the same as in the UI controller right?
Probably best to create a `ServiceUpdater` service object and use it in API and UI

Reply:
@hallelujah UI in a following PR 😄 And I don't mind doing that refactoring in that one 😄

Reply:
In particular in https://github.com/3scale/porta/pull/1084 😄 But it isn't really started... I am only testing a weird situation there first 😄